common-java-libs/org.wso2.apk.apimgt.impl/build.gradle

```diff
@@ -15,6 +15,10 @@
  *
  */
 
+plugins {
+    id 'com.avast.gradle.docker-compose' version '0.16.8'
```

Comment:
Shall we add this to `lib.versions.toml`?

Reply:
I think it's better to wrap docker-compose commands in `common-gradle-scripts/docker.gradle` and use here. @RakhithaRR WDYT?

Reply:
This docker-compose plugin is used only for testing in java right? `docker.gradle` contains tasks that are used to build and push the images. So I'm not sure whether its correct to add this plugin there.

Reply:
Yes this plugin is only used for testing